schema-guide.md

```diff
@@ -323,7 +323,7 @@ that case, `doi` can be used as shorthand for something like:<br><br>
 
 - **type**: (Array of) [`definitions.license-enum`](#definitionslicense-enum).
 - **required**: `false`
-- **description**: The SPDX license identifier(s) for the license(s) under which the work is made available. When there are multiple
+- **description**: The [SPDX license identifier(s)](https://spdx.dev/ids/) for the license(s) under which the work is made available. When there are multiple
```

Comment:
Note that over time, the list in this link will become increasingly out of sync with our list though. In the schema, we have a note which permalinks to github, maybe that's clear enough

Reply:
Decided that this is okay for now during a review call.